### PR TITLE
Commercials

### LINKED ISSUE #55: Draw hockey border and ads
See #52 for resources.

### LINKED ISSUE #59: Crowd

### PR DESCRIPTION
**Pull request by FredTheDino**
Commercials and a crowd - the crowd needs to be drawn in a different place though.
Gonna sleep right now. Nigh

### CODE REVIEW

**Comment by sornas** [comment]
This takes a really (really) long time for the parser to parse.

**Comment by sornas** [comment]
`main` is basically instant so something's off on this branch.

File: arena_mod.sy [hunk 1]
```diff
@@ -10,20 +10,74 @@ Arena :: blob {
...
+            l_gfx_sprite' assets.ads, (0, 0), pos + (t - 4.0 * scale, (scale - 0.2) / 1.5), (4.0, 4.0 * 20.0 / 401.0) * scale
+
+            l_gfx_rect' -scale * 1.3, 0.0, scale, 3.0, gray
+            l_gfx_rect' scale * 1.3, 0.0, scale, 3.0, gray
+
             l_gfx_sprite' assets.arena, (0, 0), pos, (2.0, 1.0) * scale
+            l_gfx_sprite' assets.border, (0, 0), pos + (0.0, scale / 2.0), (2.0, 2.0 * 40.0 / 200.0) * scale
+            // TODO(ed): Move this to a custom file - it needs to be
+            // rendered on top of the player
+            t :: l_time'
```
Comment (FredTheDino) [inline arena_mod.sy]: The problem is right here!

**Comment by FredTheDino** [comment]
> `main` is basically instant so something's off on this branch.

It's A LOT faster if you run in release mode. And I was wrong with my previous comment - something else is up.

**Review by sornas — APPROVED** [review]

**Pull request merged**